### PR TITLE
feat(multiselect): option to change checkmarks

### LINKED ISSUE #418: Add an option to change checkmarks in the interactive multiselect
I've run into an issue where Windows console isn't able to render the default multiselect's checkmarks ("✓" and "✗") due to them being undefined in the standard "Consolas" font.  I don't really feel like changing a font nor manually changing anything in the PTerm's source code. An option to define them would be a great addition to this awesome project.

### PR DESCRIPTION
**Pull request by KarolosLykos**
### Description
Added the option to define the checkmarks in the interactive multiselect printer.
Also added some basic test cases.

- [ ] Bug Fix
- [x] New Feature
- [x] Documentation
- [x] Other

### Related Issue
Fixes #418

### To-Do Checklist
- [x] I tested my changes
- [x] I have commented every method that I created/changed
- [x] I updated the examples to fit with my changes
- [x] I have added tests for my newly created methods

### CODE REVIEW

File: interactive_multiselect_printer.go [hunk 1]
```diff
@@ -25,9 +26,18 @@ var (
...
+			NotSelected: "✗",
+		},
 	}
 )
 
+type Checkmarks struct {
+	Selected    string
+	NotSelected string
+}
+
```
Comment (MarvinJWendt) [inline interactive_multiselect_printer.go:40]: Could you please rename this to `Checkmark` (singular) and put it into the file `/atoms.go`, it's dedicated to those small helper structs :)

File: interactive_multiselect_printer.go [hunk 2]
```diff
@@ -95,6 +106,12 @@ func (p InteractiveMultiselectPrinter) WithKeyConfirm(keyConfirm keys.KeyCode) *
 	return &p
 }
 
+// WithCheckmarks sets the checkmark
+func (p InteractiveMultiselectPrinter) WithCheckmarks(checkmarks Checkmarks) *InteractiveMultiselectPrinter {
```
Comment (MarvinJWendt) [inline interactive_multiselect_printer.go:110]: This should then be `WithCheckmark`

File: interactive_multiselect_printer.go [hunk 3]
```diff
@@ -25,9 +26,18 @@ var (
...
+		Checkmarks: Checkmarks{
+			Selected:    "✓",
+			NotSelected: "✗",
+		},
 	}
 )
 
+type Checkmarks struct {
+	Selected    string
+	NotSelected string
```
Comment (MarvinJWendt) [inline interactive_multiselect_printer.go:38]: Please rename those to `Checked` and `Unchecked`.

File: interactive_multiselect_printer.go [hunk 4]
```diff
@@ -39,6 +49,7 @@ type InteractiveMultiselectPrinter struct {
 	Selector       string
 	SelectorStyle  *Style
 	Filter         bool
+	Checkmarks     Checkmarks
```
Comment (MarvinJWendt) [inline interactive_multiselect_printer.go:52]: This should also be `Checkmark` then.

File: interactive_multiselect_printer_test.go [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+	"testing"
+
+	"atomicgo.dev/keyboard"
+	"atomicgo.dev/keyboard/keys"
+	"github.com/MarvinJWendt/testza"
+
+	"github.com/pterm/pterm"
+)
+
+func TestInteractiveMultiselectPrinter_Show(t *testing.T) {
```
Comment (MarvinJWendt) [inline interactive_multiselect_printer_test.go:13]: Thanks for adding tests! We somehow totally missed that for this printer @floaust 😅

**Review by MarvinJWendt — CHANGES_REQUESTED** [review]
Thanks for the PR, really appreciate it. Just some nitpicking stuff, otherwise it looks good 👍

**Review by MarvinJWendt — APPROVED** [review]
LGTM! Thanks for contributing 🎉 🚀

**Comment by MarvinJWendt** [comment]
I have made some additional changes to enable changing the checkmarks via a custom theme. Should be pushed soon. :)

**Comment by KarolosLykos** [comment]
> I have made some additional changes to enable changing the checkmarks via a custom theme. Should be pushed soon.

Yes, I was thinking adding that myself. That's great!!!

**Pull request merged**